### PR TITLE
Expose postgres url and all addresses as DotDict

### PR DESCRIPTION
**Pull request by andreidan**

### CODE REVIEW

File: cr8/run_crate.py [hunk 1]
```diff
@@ -295,9 +298,15 @@ def start(self):
 
     def _set_addr(self, protocol, addr):
         log.info('{0:10}: {1}'.format(protocol.capitalize(), addr))
+        host, port = addr.rsplit(':', 1)
+        port = int(port)
+        self.addresses[protocol] = Address(host, port) 
         if protocol == 'http':
             self.http_host = addr
             self.http_url = 'http://' + addr
+        elif protocol == 'psql':
```
Comment (mfussenegger) [inline cr8/run_crate.py]: I think we should not set `postgres_host` and `postgres_url` too. I'd consider `http_host` and `http_url` legacy and have `addresses` be the thing that API users should use. (Here there would also be the problem that we fix the users to `crate` which might not be the thing a client wants to have)

File: cr8/run_crate.py [hunk 2]
```diff
@@ -262,6 +263,7 @@ def start(self):
             logfile,
             self.data_path
         )
+        self.addresses = {}
```
Comment (mfussenegger) [inline cr8/run_crate.py]: I think the DotDict could already be created at this point?

File: cr8/run_crate.py [hunk 3]
```diff
@@ -295,6 +297,9 @@ def start(self):
 
     def _set_addr(self, protocol, addr):
         log.info('{0:10}: {1}'.format(protocol.capitalize(), addr))
+        host, port = addr.rsplit(':', 1)
+        port = int(port)
+        self.addresses[protocol] = Address(host, port) 
         if protocol == 'http':
             self.http_host = addr
```
Comment (andreidan) [inline cr8/run_crate.py]: @mfussenegger is there a way to mark this as deprecate? or will it be in the release notes for the next version?

**Comment by mfussenegger** [comment]
Merged via https://github.com/mfussenegger/cr8/commit/3cbe62af444fba1703ddc04a13e90fc78042c8df

File: cr8/run_crate.py (hunk 3 above)
Comment (mfussenegger) [reply cr8/run_crate.py]: I'll take care of this later

**Pull request closed without merging**